tools/run_tests/xds_k8s_test_driver/config/url-map.cfg

```diff
@@ -1,6 +1,5 @@
---resource_suffix=interop-psm-url-map
+--resource_prefix=interop-psm-url-map
 --strategy=reuse
-# TODO(lidiz): Remove the next line when xds port randomization supported.
 --server_xds_port=8848
```

Comment:
You don't need to randomize the `server_xds_port`, because the server is shared between run, right?

Reply:
Yep, that's right.